staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -0,0 +1,336 @@
+			flowcontrol.PriorityLevelConfigurationSpec{
+				Type: flowcontrol.PriorityLevelEnablementLimited,
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
+					LimitResponse: flowcontrol.LimitResponse{
+						Type: flowcontrol.LimitResponseTypeQueue,
+						Queuing: &flowcontrol.QueuingConfiguration{
+							Queues:           64,
+							HandSize:         6,
+							QueueLengthLimit: 1000,
+						},
+					},
+				},
+			}),
+		// workload-high priority-level
+		newPriorityLevelConfiguration("workload-high",
+			flowcontrol.PriorityLevelConfigurationSpec{
+				Type: flowcontrol.PriorityLevelEnablementLimited,
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
+					LimitResponse: flowcontrol.LimitResponse{
+						Type: flowcontrol.LimitResponseTypeQueue,
+						Queuing: &flowcontrol.QueuingConfiguration{
+							Queues:           128,
+							HandSize:         6,
+							QueueLengthLimit: 100,
+						},
+					},
+				},
+			}),
+		// workload-low priority-level
+		newPriorityLevelConfiguration("workload-low",
+			flowcontrol.PriorityLevelConfigurationSpec{
+				Type: flowcontrol.PriorityLevelEnablementLimited,
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
+					LimitResponse: flowcontrol.LimitResponse{
+						Type: flowcontrol.LimitResponseTypeQueue,
+						Queuing: &flowcontrol.QueuingConfiguration{
+							Queues:           128,
+							HandSize:         6,
+							QueueLengthLimit: 100,
+						},
+					},
+				},
+			}),
+		// catch-all priority-level
+		DefaultPriorityLevelConfigurationDefault,
+		// exempt priority-level
+		DefaultPriorityLevelConfigurationExempt,
+	}
+}
+
+// PredefinedFlowSchemas returns instances of the FlowSchema
+// objects shown in the Example Configuration section of the KEP
+func PredefinedFlowSchemas() []*flowcontrol.FlowSchema {
+	verbAll := []string{flowcontrol.VerbAll}
+	apiGroupAll := []string{flowcontrol.APIGroupAll}
+	resourceAll := []string{flowcontrol.ResourceAll}
+	namespaceEvery := []string{flowcontrol.NamespaceEvery}
+	resourceRuleAll := resourceRule(verbAll, apiGroupAll, resourceAll, namespaceEvery, true)
+	nonResourceRuleAll := nonResourceRule(verbAll, []string{flowcontrol.NonResourceAll})
+	return []*flowcontrol.FlowSchema{
+		newFlowSchema(
+			"system-nodes", "system", 2500,
+			flowcontrol.FlowDistinguisherMethodByUserType,
+			flowcontrol.PolicyRulesWithSubjects{
+				Subjects:         groups(user.NodesGroup),
+				ResourceRules:    []flowcontrol.ResourcePolicyRule{resourceRuleAll},
+				NonResourceRules: []flowcontrol.NonResourcePolicyRule{nonResourceRuleAll},
+			},
+			flowcontrol.PolicyRulesWithSubjects{
+				Subjects: kubeSystemServiceAccount(flowcontrol.NameAll),
+				ResourceRules: []flowcontrol.ResourcePolicyRule{
+					resourceRule(verbAll, []string{corev1.GroupName}, []string{"endpoints", "configmaps"}, namespaceEvery, false),
```

Comment:
Copy paste error? This doesn't look like node traffic.

Reply:
fixed